contrib/opencensus-ext-django/README.rst

```diff
@@ -40,6 +38,30 @@ for a complete reference.
             )''',
         }
     }
+    
+Additional configuration can be provided, please read
```

Comment:
I would put these lines `Additional configuration...` at the end.

Reply:
Done